synth.py

```diff
@@ -357,13 +357,18 @@ def from_service_account_file\(cls, filename, \*args, \*\*kwargs\):
 # ----------------------------------------------------------------------------
 # Add templated files
 # ----------------------------------------------------------------------------
-templated_files = common.py_library(unit_cov_level=78, cov_level=78)
+templated_files = common.py_library(cov_level=76)
```

Comment:
This # seems to have dropped in a regen. I didn't commit the library code changes as I'd prefer they come in through Autosynth.